lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1742,6 +1742,10 @@ sub store_column {
         }
         # make sure no modules are left running
         $self->modules->search({result => RUNNING})->update({result => NONE});
+
+        for my $module (OpenQA::Schema::Result::JobModules::job_modules($self)) {
```

Comment:
The function `OpenQA::Schema::Result::JobModules::job_modules` is likely part of the wrong package. I guess that should be improved at some point (not within this PR).